src/locales/fr.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Notes": {
+    "Add": {
+      "add_note": "Ajouter une note"
+    },
+    "EditorLoading": {
+      "back_to_list": "Retour à la liste"
+    },
+    "EditorView": {
+      "main_placeholder": "Écrivez ce que vous souhaitez…"
+    },
+    "Editor": {
+      "title_placeholder": "Ici votre titre"
+    }
+  },
+  "Error": {
+    "unshared_title": "The partage n'est plus actif",
```

Comment:
s/The/Le/

Reply:
https://github.com/cozy/cozy-notes/pull/10